lib/cell.rb

```diff
@@ -0,0 +1,40 @@
+class Cell
+
+    attr_reader :status
+
+    def initialize
+        @status = :empty
+    end
+
+    def empty?
+        if @status == :empty
+            true
+        else
+            false
+        end
+    end
+
```

Comment:
You can simplify 10-14 to one line of code because `@status == :empty` returns a boolean.